### PR TITLE
recon-ng 4.8.3 (new formula)

### PR DESCRIPTION
**Pull request by knqyf263**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/recon-ng.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+class ReconNg < Formula
+  include Language::Python::Virtualenv
+
+  desc "The Web Reconnaissance Framework"
```
Comment (bfontaine) [inline Formula/recon-ng.rb]: You can drop `The` here: `desc "Web Reconnaissance Framework"`.

File: Formula/recon-ng.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    venv.pip_install resources
+
+    # Replace shebang with virtualenv python
+    inreplace libexec/"recon-ng", "#!/usr/bin/env python", "#!#{libexec}/bin/python"
+
+    bin.install_symlink libexec/"recon-ng"
+  end
+
+  test do
+    system "#{bin}/recon-ng", "-v"
```
Comment (bfontaine) [inline Formula/recon-ng.rb]: Can’t we write a test that does something more substantial than e.g. `-v` or `-h`?

File: Formula/recon-ng.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    sha256 "e28f902f2f0a1603ea95ebe21dff311ef09be3d0f0ef29a3e44a932729564385"
+  end
+
+  def install
+    libexec.install Dir["*"]
+    venv = virtualenv_create(libexec)
+    venv.pip_install resources
+
+    # Replace shebang with virtualenv python
+    inreplace libexec/"recon-ng", "#!/usr/bin/env python", "#!#{libexec}/bin/python"
```
Comment (bfontaine) [inline Formula/recon-ng.rb]: Is this necessary?

**Review by bfontaine — DISMISSED** [review]

File: Formula/recon-ng.rb (hunk 1 above)
Comment (knqyf263) [inline Formula/recon-ng.rb]: Thanks! I'll fix.

File: Formula/recon-ng.rb (hunk 2 above)
Comment (knqyf263) [inline Formula/recon-ng.rb]: I added a little better test.

File: Formula/recon-ng.rb (hunk 3 above)
Comment (MikeMcQuaid) [inline Formula/recon-ng.rb]: @tdsmith May know if there's a better way.

Comment (tdsmith) [inline Formula/recon-ng.rb]: This looks reasonable.

Comment (knqyf263) [inline Formula/recon-ng.rb]: @tdsmith Thanks for your review!
@bfontaine I fixed all your requested changes. 
Check again if there is no problem, please.

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**